test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +39,33 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNullTest() throws Exception {
+        assertFalse(isAnyNull());
+        assertFalse(isAnyNull(1, 2));
+        assertTrue(isAnyNull(null, null));
+        assertTrue(isAnyNull(1, 2, null));
+    }
+
+    @Test
+    public void nameSimilarityTest() throws Exception {
+        Name name = new Name("John Smith");
+        // Two identical names should be similar.
+        assertTrue(name.isSimilar(new Name("John Smith")));
+
+        // Two very different names should not be similar.
+        assertFalse(name.isSimilar(new Name("Toodles McDoodles")));
+
+        // A name with one different letter should be considered as similar.
+        assertTrue(name.isSimilar(new Name("John Smitt")));
+
+        // A name with two different letters should be considered as similar.
+        assertTrue(name.isSimilar(new Name("Jhon Smith")));
+
+        // A name with three different letters should not be considered as similar.
+        assertFalse(name.isSimilar(new Name("Shon Smith")));
+    }
+
```

Comment:
I think your tests are very easy to follow and are good. Proper testing would be covered later in the sem. But I just wanted to bring up some other corner cases that you could have considered. For example testing with `""` or with `null` or not restricting your cases to the format `"John Smith".isSimilar(___)`. The point of tests is to cover as many corner cases and possible pitfalls as possible.